zeppelin-zengine/pom.xml

```diff
@@ -42,6 +42,18 @@
     </dependency>
 
     <dependency>
+      <groupId>${project.groupId}</groupId>
```

Comment:
I agree with you. But To use HDFSCommand functions, I had to add dependency.
Could you give me advice to solve this?